README.md

```diff
@@ -109,57 +129,68 @@ editing `env_run.xml` in the case directory to change `DOUT_S` from `FALSE` to `
 
 Please see the [wiki](https://github.com/ESCOMP/CAM/wiki) for complete documentation on CAM, getting started with git and how to contribute to CAM's development.
 
-### _FTorch_ on Derecho
+### `FTorch` on Derecho
 
-The following steps can be followed to ensure FTorch is built to be consistent
+The following steps can be followed to ensure `FTorch` is built to be consistent
 with CAM on Derecho.
 
-On Derecho `libtorch` should be loaded using
-```
-module load libtorch/2.1.2
-```
-and used to build _FTorch_.\
+#### load CAM environment
 
-Further, for compatibility with CAM we need to be specific about the environment and
-compilers we load.
-The following sequence of modules are required to build FTorch compatible with the intel
-build of CAM on Derecho:
-```
+For compatibility with CAM we need to be specific about the environment and compilers we load. The following sequence of modules
+are required to build `FTorch` compatible with the intel build of CAM on Derecho:
+
+```bash
 module purge
 module load ncarenv/23.06
 module load intel-oneapi/2023.0.0
 module load mkl
 module load cmake
-module load libtorch/2.1.2
 module load cuda/11.7.1
 ```
-Note that in future builds or releases, or on different machines, the environment for
-building CAM may change.
-In this case the FTorch environment should be updated accordingly.
 
-FTorch can then be built and installed from `/path/to/ftorch/src/build/` as described in the
-documentation with:
+> [!NOTE]
+> In future builds or releases, or on different machines, the environment for building CAM may change. In this case the `FTorch`
+> environment should be updated accordingly.
+
+> [!NOTE]
+> When building `FTorch` with `cmake` in the next step, we use the absolute path to `libtorch`. Previously, we loaded the
+> `libtorch` module i.e., `module load libtorch/2.1.2` but this conflicts with `ncarenv/23.06`. We do not actually need to load
+> the `libtorch` module to use the absolute path, so for now we can ignore this.
+
```

Comment:
I wonder if it is just easier to not mention this to avoid potential confusion?
"Previously" isn't really relevant here as to my knowledge no-one from NCAR/DataWave has yet followed these instructions.

Reply:
removed in [269a5d0](https://github.com/DataWaveProject/CAM/pull/19/commits/269a5d0b592b1285b415601819a42795128ade76)